Cargo.toml

```diff
@@ -102,13 +102,6 @@ slow-blocks = ["ethcore/slow-blocks"]
 secretstore = ["parity-secretstore", "accounts", "ethabi", "ethcore-call-contract"]
 final = ["parity-version/final"]
 deadlock_detection = ["parking_lot/deadlock_detection"]
-# to create a memory profile (requires nightly rust), use e.g.
-# `heaptrack /path/to/parity <parity params>`,
-# to visualize a memory profile, use `heaptrack_gui`
-# or
-# `valgrind --tool=massif /path/to/parity <parity params>`
-# and `massif-visualizer` for visualization
-memory_profiling = []
```

Comment:
//cc @ordian 

This doesn't seem to with `parity-util-mem` because it configures `global jemalloc` so I removed it, thoughts?
Maybe, we do something inside `parity-util-mem` instead (but quite annoying to use here here with additive feature flags?

Reply:
ah, right, it's outdated now, let's remove it and for memory profiling we would need to modify https://github.com/paritytech/parity-ethereum/blob/ff78f4318ab2c51063622ef32cd15f9ce62d95d3/Cargo.toml#L62